locales/translations/es.json

```diff
@@ -451,6 +451,7 @@
     "Response must not be blank": "La respuesta no debe estar en blanco",
     "Responses": "Respuestas",
     "Responsive": "Responsivo",
+    "Restart simulation": "",
```

Comment:
Missing spanish translation of the button.

Reply:
It should be something like: _Reiniciar simulación_ or _Reiniciar la simulación_
@diegoliberman, who should approve this translations?

Reply:
Let's translate what we feel confident ourselves.  "Reiniciar simulación" sounds good.  If we come up with a case that we don't know how to translate, we'll ask Manu.